phoenix-core/src/main/java/org/apache/phoenix/trace/PhoenixMetricsSink.java

```diff
@@ -175,7 +176,9 @@ private void createTable(Connection conn, String table) throws SQLException {
                         TAG_COUNT + " smallint, " +
                         ANNOTATION_COUNT + " smallint" +
                         "  CONSTRAINT pk PRIMARY KEY (" + TRACE.columnName + ", "
-                        + PARENT.columnName + ", " + SPAN.columnName + "))\n";
+                        + PARENT.columnName + ", " + SPAN.columnName + "))\n" +
+                        PhoenixDatabaseMetaData.TRANSACTIONAL + "=" + Boolean.FALSE;
```

Comment:
Is the intent here to make tracing table transactional? What would happen to the existing tracing table i.e. on a cluster with existing phoenix deployment with tracing enabled? You will likely need some kind of upgrade path to update the metadata of existing tracing table to mark it as transactional.